specs/current/nexu-pal.md

```
@@ -47,14 +88,29 @@ Both workflows create a short-lived token via `actions/create-github-app-token@v
 | `NEXU_PAL_PRIVATE_KEY_PEM` | GitHub App private key |
 | `OPENAI_BASE_URL` | OpenRouter base URL |
 | `OPENAI_API_KEY` | OpenRouter API key |
+| `ISSUE_SYNC_FEISHU_BOT_WEBHOOK` | Feishu incoming webhook URL for the existing issue-opened and discussion notifications |
+| `ISSUE_TRIAGE_BUG_FEISHU_WEBHOOK` | Feishu incoming webhook URL for bug triage notifications |
+| `ISSUE_TRIAGE_REQ_FEISHU_WEBHOOK` | Feishu incoming webhook URL for non-bug triage notifications |
 
 ## File map
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)